packages/editor-ui/src/plugins/codemirror/resolvableHighlighter.ts

```diff
@@ -63,29 +63,31 @@ const coloringStateField = StateField.define<DecorationSet>({
 		return Decoration.none;
 	},
 	update(colorings, transaction) {
-		colorings = colorings.map(transaction.changes); // recalculate positions for new doc
+		try {
+			colorings = colorings.map(transaction.changes); // recalculate positions for new doc
 
-		for (const txEffect of transaction.effects) {
-			if (txEffect.is(coloringStateEffects.removeColorEffect)) {
-				colorings = colorings.update({
-					filter: (from, to) => txEffect.value.from !== from && txEffect.value.to !== to,
-				});
-			}
+			for (const txEffect of transaction.effects) {
+				if (txEffect.is(coloringStateEffects.removeColorEffect)) {
+					colorings = colorings.update({
+						filter: (from, to) => txEffect.value.from !== from && txEffect.value.to !== to,
+					});
+				}
 
-			if (txEffect.is(coloringStateEffects.addColorEffect)) {
-				colorings = colorings.update({
-					filter: (from, to) => txEffect.value.from !== from && txEffect.value.to !== to,
-				});
+				if (txEffect.is(coloringStateEffects.addColorEffect)) {
+					colorings = colorings.update({
+						filter: (from, to) => txEffect.value.from !== from && txEffect.value.to !== to,
+					});
 
-				const decoration = resolvableStateToDecoration[txEffect.value.state ?? 'pending'];
+					const decoration = resolvableStateToDecoration[txEffect.value.state ?? 'pending'];
 
-				if (txEffect.value.from === 0 && txEffect.value.to === 0) continue;
+					if (txEffect.value.from === 0 && txEffect.value.to === 0) continue;
 
-				colorings = colorings.update({
-					add: [decoration.range(txEffect.value.from, txEffect.value.to)],
-				});
+					colorings = colorings.update({
+						add: [decoration.range(txEffect.value.from, txEffect.value.to)],
+					});
+				}
 			}
-		}
+		} catch (error) {}
```

Comment:
Nit: Is this really ignorable error? Should we notify sentry or print to console?

Reply:
Yeah, that's the way. Sentry is handled by the frontend hooks atm and we are not using it that much in-app